src/zh/zazhimi/src/eu/kanade/tachiyomi/extension/zh/zazhimi/Response.kt

```diff
@@ -0,0 +1,70 @@
+package eu.kanade.tachiyomi.extension.zh.zazhimi
+
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SManga
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class IndexResponse(
```

Comment:
Don't need to use `data class`
```suggestion
class IndexResponse(
```